.travis.yml

```diff
@@ -38,11 +41,21 @@ jobs:
       script: /bin/bash travis/docker-functions.sh docker_build $VERSION publish
       if: branch = env(DEV_BRANCH) AND type = push
       node_js: 10
+    - stage: publish
+      name: openapi typescript client alpha
+      script: bash client/generate.sh publish
+      if: branch = env(DEV_BRANCH) AND type = push
+      node_js: 10
     - stage: release
       name: docker publish release
       script: /bin/bash travis/docker-functions.sh docker_build $VERSION release
       if: branch = env(RELEASE_BRANCH) AND type = api AND commit_message = env(RELEASE_MESSAGE)
       node_js: 10
+   -  stage: release
```

Comment:
there are 2 spaces in  `-  stage`, you need to delete one.

Reply:
Fixed, using prettier to restyle